src/libDirectoryService/DSBlockPostProcessing.cpp

```diff
@@ -365,13 +365,6 @@ void DirectoryService::StartNextTxEpoch() {
   ResetPoWSubmissionCounter();
   m_viewChangeCounter = 0;
 
-  {
```

Comment:
Reason for removal?

Reply:
these buffer as such are supposed to be cleared at start of first tx epoch only.
while this func `StartNextTxEpoch` is being invoked after rejoin.